pkg/rpc/connection_class.go

```diff
@@ -61,36 +59,18 @@ func (c ConnectionClass) String() string {
 // SafeValue implements the redact.SafeValue interface.
 func (ConnectionClass) SafeValue() {}
 
-var systemClassKeyPrefixes = []roachpb.RKey{
-	roachpb.RKey(keys.Meta1Prefix),
-	roachpb.RKey(keys.NodeLivenessPrefix),
-}
-
-// isSystemKey returns true if the given key belongs to a range eligible for
-// SystemClass connection.
-//
-// Generally, not all system ranges are eligible. For example, the timeseries
-// ranges are not, because they can be busy and disrupt other system traffic. We
-// try to make SystemClass responsive by keeping it small.
```

Comment:
Please retain some of this comment, because keeping the system class small is an important goal.

Reply:
retained